src/services/user.services.js

```diff
@@ -43,6 +43,17 @@ async function UpdatePassword(email, pass) {
 async function deleteUser(id) {
   return User.destroy({ where: { id } });
 }
+async function disableOtp(id) {
+  const findData = await User.findByPk(id);
+  findData.tfa_enabled = false;
+  await findData.save().then((result) => result);
```

Comment:
what's the purpose of .then() here?

Reply:
the purpose of of this result method is to return the results of `save()` after it has completed.